src/server/ProcessHandle.cpp

```diff
@@ -35,12 +35,12 @@ ConsoleProcessHandle::ConsoleProcessHandle(const DWORD dwProcessId,
     }
 }
 
-CD_CONNECTION_INFORMATION ConsoleProcessHandle::GetConnectionInformation() const
+CD_CONNECTION_INFORMATION ConsoleProcessHandle::GetConnectionInformation(IDeviceComm* deviceComm) const

```

Comment:
This might warrant a doc comment now too

Reply:
Docced!